### PR TITLE
Added support for numpy values (#24)

### PR DESCRIPTION
**Pull request by fraimondo**

### CODE REVIEW

File: h5io/_h5io.py [hunk 1]
```diff
@@ -151,8 +151,9 @@ def _triage_write(key, value, root, comp_kw, where,
         else:  # isinstance(value, float):
             title = 'float'
         _create_titled_dataset(root, key, title, np.atleast_1d(value))
-    elif isinstance(value, np.bool_):
-        _create_titled_dataset(root, key, 'np_bool_', np.atleast_1d(value))
+    elif isinstance(value, (np.integer, np.floating, np.bool_)):
+        title = 'np_{}'.format(value.__class__.__name__)
```
Comment (larsoner) [inline h5io/_h5io.py]: change to `{0}` for Python2.6

**Comment by fraimondo** [comment]
Hi @larsoner ,

Any reason for not merging this?

I was using MNE and came across this issue again.

**Comment by larsoner** [comment]
Thanks @fraimondo. Can you open a PR to MNE to update `mne/externals/h5io.py`?

**Comment by fraimondo** [comment]
Done!

https://github.com/mne-tools/mne-python/pull/4721

**Pull request merged**